float32 v_x
float32 v_y
float32 v_pitch
float32 roll
float32 grasp
int32 dnf_x
int32 dnf_y